### DISCUSSION TITLE
Css not activating when html is

### DISCUSSION THREAD

**Discussion by Spoednik1** [community/community#21419] [Other Feature Feedback, Questions, & Ideas]
<p>Hey fellas i am new to github and the everything related. I was following some YT tutorials on how to upload files and such. I did as the vid had shown but my css does not seem to be activated. Honest help is appreciated. this is mine   <a href="https://github.com/Spoednik1/test-web" class="inline-onebox" rel="noopener nofollow ugc">GitHub - Spoednik1/test-web: testing</a></p>

**Comment by jfullstackdev** [comment]
<p>it’s because of the URL, <code>Css\styles.css</code>, you should change this to <code>styles.css</code></p>
<p><img src="https://user-images.githubusercontent.com/47092464/181064497-74e85e10-de4e-407d-800f-1b08da2c3080.png" alt="image" data-base62-sha1="cc2GSkdVBtBQPii5WYICylWaqyk" width="399" height="58"></p>

**Comment by Spoednik1** [comment]
<p>Thank you Thank you. It worked. Sorry for the simple mistake.</p>

**Comment by Spoednik1** [comment]
<p>I apologize again for the simple mistake, but I thought that fixing the CSS would fix the images into making them showing up, but they arent. Any suggestions?</p>

**Comment by Spoednik1** [comment, marked as answer]
<p>No worries I tried around and figured it out thanks anyways!</p>
